hindiCG.txt

```diff
@@ -0,0 +1,72 @@
+ DELIMITERS = "<$।>" ;
+ SUBREADINGS = LTR ;
+
+LIST POST = post;
+LIST MADJ = (adj m);
+LIST NOUN = n;
+LIST FADJ = (adj f);
+LIST NOM = (n nom);
+LIST FN = (n f );
+LIST MN = (n m );
+LIST VBSER = vbser;
+LIST CNJ = cnjcoo;
+LIST ADJ = adj;
+LIST OBL = (n obl);
+LIST ADV = adv;
+
+SECTION
+
+#1 
+SELECT ADV IF 
+	(0 ADJ )
+	(1 ADJ );
+#2
+SELECT ADJ IF 
+	(0 ADJ ) 
+	(0 NOUN ) 
+	(1 NOUN );
+#3
+SELECT FADJ 
+	(1 FN );
+
+#4
+SELECT MADJ  IF
+	(1 MN );
+
+#5
+SELECT NOM IF
+	(1 ADJ ) 
+	(2 VBSER );
+
+#6
+SELECT OBL IF 
+	(1 CNJ )
+ 	(2 NOUN ) 
+	(3 POST );
+
+#7
+SELECT OBL IF 
+	(1 POST );
+
+#8
+SELECT FADJ IF
+	(-1 FN );
```

Comment:
You could make the names 'FemaleNoun' etc. instead of 'FN' for readability. As more CG rules get added it becomes hard to keep track of everything. Also note that although the documentation has all-caps, 'select' is the same as 'SELECT' for CG and is arguably more readable.

Reply:
Edited this